src/components/MyKiva/LatestBlogCarousel.vue

```diff
@@ -19,7 +19,7 @@
 					:title="card.title"
 					:date="card.date"
 					:slug="card.slug"
-					@track="trackBlogCard"
+					@track="() => trackBlogCard(card)"
```

Comment:
normally you wouldn't need the `() => ` portion here, but it shouldn't hurt anything.